pennylane/tape/qscript.py

```diff
@@ -889,44 +889,96 @@ def expand(self, depth=1, stop_at=None, expand_measurements=False):
             expand_measurements (bool): If ``True``, measurements will be expanded
                 to basis rotations and computational basis measurements.
 
-        **Example**
+        .. seealso:: :func:`~.pennylane.devices.preprocess.decompose` for a transform that
+           performs the same job and fits into the current transform architecture.
 
-        Consider the following nested quantum script:
+        .. warning::
 
-        >>> nested_script = QuantumScript([qml.Rot(0.543, 0.1, 0.4, wires=0)])
-        >>> ops = [
-                qml.BasisState(np.array([1, 1]), wires=[0, 'a']),
-                nested_script,
-                qml.CNOT(wires=[0, 'a']), qml.RY(0.2, wires='a'),
-            ]
-        >>> measurements = [qml.probs(wires=0), qml.probs(wires='a')]
-        >>> qscript = QuantumScript(ops, measurements)
+            This method cannot be used with a tape with non-commuting measurements, even if
+            ``expand_measurements=False``.
 
-        The nested structure is preserved:
+            >>> tape = qml.tape.QuantumScript([], [qml.expval(qml.X(0)), qml.expval(qml.Y(0))])
```

Comment:
This line is too long for the `warning` box, even at full screen width it's pretty narrow. So are the two `QuantumFunctionError` outputs below.

Reply:
shrinking some line lengths now. thanks for the sugggestion.